### DISCUSSION TITLE
Is Viomi SE viomi.vacuum.v19 really supported?

### DISCUSSION THREAD

**Discussion by rufik** [Hypfer/Valetudo#1692] [Q&A / Support] [answered]
Supported devices (https://valetudo.cloud/pages/general/supported-robots.html#viomi_se) page shows that Viomi SE is supported, but rooting instructions page (https://github.com/Hypfer/valetudo-crl200s-root) does not list `viomi.vacuum.v19` model, only `v7`.
So is that `v19` model really supported?

**Comment by Hypfer** [comment, marked as answer]
The readme of https://github.com/Hypfer/valetudo-crl200s-root has been updated to be more clear on this.

**Reply by rufik** [reply]
Very nice, thank you. It's stated "_Please refer to the docs on Valetudo.cloud to find a full list of all supported models_", so this model **is** supported :)
